web_src/less/_organization.less

```diff
@@ -58,6 +58,10 @@
         }
       }
 
+      #rss-feed {
+        transform: translate(0, calc(50% - 18px));
```

Comment:
Can there a solution other than `transform: translate`? Although the result is correct, personally I feel it a little strange to use it here.

Reply:
@wxiaoguang Do you have any ideas how else it can be done?

Using
```css
      .feed-button {
        display: flex;
        align-items: center;
      }
```

Works to align the SVG within the `<i>`, but it gets placed on a new line because of the `display: flex`.

Reply:
I haven't tried that, if it is the only choice, I am fine too

Reply:
I've already tried this and some other stuff, and this seems to be the only thing that works.

Reply:
`flex` layout is indeed the best choice in my mind.

Reply:
I was able to get it to work by putting the `display: flex` and `align-items: center` in the `.ui.header` block.